plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -139,7 +163,66 @@ class Bazel(override val descriptor: PluginDescriptor = BazelFactory.descriptor)
         val moduleMetadata = Parser(definitionFile.readText()).parse()
         val depDirectives = moduleMetadata.dependencies.associateBy { "${it.name}@${it.version}" }
         val archiveOverrides = getArchiveOverrides(projectDir)
-        val scopes = getDependencyGraph(projectDir, depDirectives, archiveOverrides)
+        var (mainModule, scopes) = getDependencyGraph(
+            projectDir,
+            depDirectives,
+            archiveOverrides
+        )
+
+        val conanExtension = mainModule.extensionUsages.firstOrNull { "conan_deps_module_extension.bzl" in it.key }
+        val conanPackages = mutableSetOf<Package>()
+
+        conanExtension?.let { extension ->
+            val conanProjectReferences = mutableSetOf<PackageReference>()
```

Comment:
How about moving this whole long block to a function for readability?

Reply:
The problem is that `resolveConanDependencies` has a lot of parameters to carry on. Do you still think this would be worthwhile ?

Reply:
I'd say it depends on the concrete number... if it's more than 7 parameters, then not, probably.